### PR TITLE
Add extra pipeline for 3scale test

### PR DESCRIPTION
**Pull request by tremes**

### CODE REVIEW

**Comment by tremes** [comment]
I decided to create an extra pipeline for our 3scale test so that we can update the w2-test-executor then.

File: jobs/integr8ly/ocp3/test-suites/3scale-test.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+
+- job:
+    name: 3scale-tests
```
Comment (pawelpaszki) [inline jobs/integr8ly/ocp3/test-suites/3scale-test.yaml]: the name indicates running number of tests. Perhaps rename it to `3scale-test`?

File: jobs/integr8ly/ocp3/test-suites/3scale-test.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            node('cirhos_rhel7'){
+
+                Boolean publishTestResults = true
+
+                stage('Clone QE repository') {
+                    dir('.') {
+                        git branch: "${BRANCH}", url: "${REPOSITORY}"
+                    }
+                }
+                stage('Run the 3scale tests'){
```
Comment (pawelpaszki) [inline jobs/integr8ly/ocp3/test-suites/3scale-test.yaml]: same here - the name of the stage indicates running number of tests. Perhaps rename it to `Run the 3scale-test`?

**Comment by pawelpaszki** [comment]
LGTM apart from the minor comments

File: jobs/integr8ly/ocp3/test-suites/3scale-test.yaml (hunk 1 above)
Comment (tremes) [reply jobs/integr8ly/ocp3/test-suites/3scale-test.yaml]: OK

**Comment by tremes** [comment]
Successful run at https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/w2-test-executor-tremes/3/

**Pull request merged**